types/jquery-jcrop/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "only-arrow-functions": false
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
OK, I was just trying to preserve the format of the tests taken from the documentation.  But I can make that change if it is important to do so.